hadoop-yarn-project/hadoop-yarn/hadoop-yarn-applications/hadoop-yarn-services/hadoop-yarn-services-core/src/test/java/org/apache/hadoop/yarn/service/TestYarnNativeServices.java

```diff
@@ -917,7 +917,8 @@ private void checkEachCompInstancesInOrder(Component component, String
 
     int i = 0;
     for (String s : instances) {
-      assertThat(s).isEqualTo(component.getName() + "-" + i);
+      assertThat(s,
+          CoreMatchers.containsString(component.getName() + "-" + i));
```

Comment:
I'm not sure if there's any side effects here, we are changing the test.

Can we stick with equals?

```suggestion
      assertThat(s, CoreMatchers.equalTo(component.getName() + "-" + i));
```

Reply:
I think your suggestion is correct and should be changed to equal.

Reply:
I think your suggestion is correct and should be changed to equal.